core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts

```diff
@@ -271,13 +271,13 @@ export function createFavoritePagesURL(params: {
 }
 
 /**
- * Create a pure URL from the params
  *
+ * @description Create a full URL with the clientHost
  * @export
  * @param {DotPageApiParams} params
  * @return {*}  {string}
  */
-export function createPureURL(params: DotPageApiParams): string {
+export function createFullURL(params: DotPageApiParams): string {
```

Comment:
We need to also send the `host_id` to support multisite

Reply:
Out of the issue's scope. It may need to be addressed in a new ticket.

Reply:
After a talk with @zJaaal, we are going to address the `host_id` here

Reply:
Done!